opm/parser/eclipse/EclipseState/Schedule/OilVaporizationProperties.hpp

```diff
@@ -63,8 +63,8 @@ namespace Opm
         bool defined() const;
         size_t numPvtRegions() const {return m_maxDRSDT.size();}
 
-        const std::vector<double>& vap1() const;
-        const std::vector<double>& vap2() const;
+        double vap1() const;
```

Comment:
Hmm - looks like we now two quite similar functions:
```
double vap1() const;
double getVap1() const;
```
that is maybe a bit excessive?

Reply:
Yeah, of course we had two accessors before, will nuke one of them.

Reply:
Actually, I either need the two accessors OR i need to disable the throw. I do not want to complicate the serialization.

Reply:
Disable the throw